packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -413,15 +428,15 @@ function UserMenu({
             >
               Organizations
             </MenuItem>
-            {!isGovernment && (
+            {!isGovernment && !!account?.hasLegacy2fa && (
```

Comment:
should this need a `!!` if hasLegacy2fa is returning a boolean?

Reply:
Nope, it doesn't - I noticed that there were places where we did `!!admin` so I wasn't sure if it was a convention that we wanted to force the value to be boolean instead of relying on truthy/falsey. I can remove this.

Reply:
👍

Reply:
I think that coercion is for cases where it's a boolean or a nullish value in those cases. If `hasLegacy2fa` is always a boolean, that's not an issue.